tests/test_jointcal_hsc.py

```diff
@@ -15,16 +15,16 @@
 import jointcalTestBase
 
 try:
-    data_dir = lsst.utils.getPackageDir('validation_data_hsc')
-    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'sdss-dr9-fink-v5b')
+    data_dir = lsst.utils.getPackageDir('testdata_jointcal')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'hsc_and_index')
 except lsst.pex.exceptions.NotFoundError:
     data_dir = None
 
 # We don't want the absolute astrometry to become significantly worse
 # than the single-epoch astrometry (about 0.040").
 # This value was empirically determined from the first run of jointcal on
 # this data, and will likely vary from survey to survey.
-absolute_error = 52e-3*u.arcsecond
+absolute_error = 53e-3*u.arcsecond
```

Comment:
The HSC tests weren't really "finalized" before, so I'm not to surprised the numbers needed tweaking once I settled on a data set and number of visits in the tests.